src/akamai/lib/Logger.as

```diff
@@ -0,0 +1,35 @@
+package
+{
+import flash.external.ExternalInterface;
+
+  public class Logger
+  {
+    /**
+     * Constructor
+     * @public
+     */
+     public function Logger( )
+     { 
+     }
+   
+    /**
+     * Send messages to the browser console log.In future this can be hooked to any other Debugging tools.
+     * @private
+     * @method ExternalJavaScriptAPI#sendToDebugger
+     * @param {string} value The value to be passed to the browser console.
+     * @param {string} referrer The fuction or process which passed the value.
+     * @param {string} channelBranch It can be info, debug, warn, error or log.
+     * @returns {boolean} True or false indicating success
+     */
+     public function sendToDebugger(value:String, referrer:String = null,
+                                    channelBranch:String = "log"):Boolean
+     {
+       // channelBranch = info, debug, warn, error, log
+       var channel:String = "OO." + channelBranch;
```

Comment:
Right now only OO.log is available.  May want to get rid of channelBranch before this causes an issue or change logic to only use OO.log when level is log or info